src/RepoToolset/DefaultVersions.props

```diff
@@ -1,30 +1,41 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Is this <xml> tag necessary? I'd rather avoid it.

Reply:
It lets editors that are not VS recognize the file as XML and behave correctly.

It is very painful to update these files otherwise.